v1/pipeline/boolean-input.yaml

```diff
@@ -0,0 +1,35 @@
+title: BooleanInput
+allOf:
+  - $ref: ./input.yaml
+  - type: object
+    required:
+      - type
+    properties:
+      default:
+        type: boolean
+      type:
+        type: string
+        enum:
+          - boolean
+      value:
+        type: boolean
+      validator:
+        type: object
+        oneOf:
+          - allOf:
+              - properties:
+                  allowed_values:
```

Comment:
is this needed for boolean wouldn't this be always fixed

Reply:
yeah, for boolean we can remove the validator, as it will only allow true and false values anyway